server/api/controllers/broker.js

```diff
@@ -169,17 +270,19 @@ const executeBrokerDeal = (req, res) => {
 		symbol,
 		price,
 		size,
```

Comment:
@fetok12 I dont see any reason for the client to send side, symbol, price and size anymore. Client sends the token which is all the server needs. It should already have the other information stored on redis. With the current logic I will get the quote then send some other price and symbol and the system will execute the broker with the price sent by user which is really really dangerous.

I also still see checking the broker buy_price sell_price comparison with the price user sent which does not make any sense.